### PR TITLE
Add steps to dump staging data on local

### PR DESCRIPTION
**Pull request by chiraggshah**

### CODE REVIEW

**Comment by chiraggshah** [comment]
@RahulMahale _a : Please review

File: docs/how-to.md [hunk 1]
```diff
@@ -105,3 +105,23 @@ check all the commits deployed on server using
 ```
 bundle exec cap production deploy:revison_all
  ```
+
+### How do I import staging dump file on local
+
+- Create a user **deployer** in psql and **caregeneral_staging** database
+```
+psql
+CREATE USER tom WITH PASSWORD 'myPassword';
```
Comment (RahulMahale) [inline docs/how-to.md]: @chiraggshah  this should  be like this,
`CREATE USER deployer WITH PASSWORD 'welcome';`
user should be deployer and not tom

File: docs/how-to.md [hunk 2]
```diff
@@ -105,3 +105,23 @@ check all the commits deployed on server using
...
+
+- Grant **deployer** user privileges and exit psql
+```
+GRANT ALL PRIVILEGES ON DATABASE caregeneral_staging to deployer;
+\q
+```
+
+- Import the .dump file into the **caregeneral_staging** database
+```
+pg_restore -i -h localhost -p 5432 -U deployer -d caregeneral_staging -v "caregeneral_production_latest.dump"
```
Comment (RahulMahale) [inline docs/how-to.md]: @chiraggshah how about pointing it to dump path like this
``` pg_restore -i -h localhost -p 5432 -U deployer -d caregeneral_staging -v "<path-to-latest-dump>" ```

**Comment by chiraggshah** [comment]
@RahulMahale : Made the suggested changes.

**Pull request merged**